### PR TITLE
Fixup link on GenerateRecoveryCodes page

### LINKED ISSUE #226: Link on Razor Pages individual auth template page GenerateRecoveryCodes is broken
There's a bug in the 2.0.5 Razor Pages individual auth template where the link in GenerateRecoveryCodes.cshtml is incorrect:

```
<p>
    Generating new recovery codes does not change the keys used in authenticator apps. If you wish to change the key
    used in an authenticator app you should <a asp-action="ResetAuthenticatorWarning">reset your authenticator keys.</a>
</p>
```

Needs to be changed to

```
<p>
    Generating new recovery codes does not change the keys used in authenticator apps. If you wish to change the key
    used in an authenticator app you should <a asp-page="./ResetAuthenticator">reset your authenticator keys.</a>
</p>
```

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes #226

### CODE REVIEW

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Account/Manage/GenerateRecoveryCodes.cshtml [hunk 1]
```diff
@@ -16,12 +16,12 @@
     </p>
     <p>
         Generating new recovery codes does not change the keys used in authenticator apps. If you wish to change the key
-        used in an authenticator app you should <a asp-action="ResetAuthenticatorWarning">reset your authenticator keys.</a>
+        used in an authenticator app you should <a asp-page="./ResetAuthenticator">reset your authenticator keys.</a>
```
Comment (pranavkm) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Account/Manage/GenerateRecoveryCodes.cshtml]: See https://github.com/aspnet/Identity/blob/dev/src/UI/Areas/Identity/Pages/Account/Manage/GenerateRecoveryCodes.cshtml#L19

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Account/Manage/GenerateRecoveryCodes.cshtml [hunk 2]
```diff
@@ -16,12 +16,12 @@
...
     </p>
 </div>
 
 <div>
-    <form asp-action="GenerateRecoveryCodes" method="post" class="form-group">
+    <form method="post" class="form-group">
```
Comment (pranavkm) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Account/Manage/GenerateRecoveryCodes.cshtml]: See https://github.com/aspnet/Identity/blob/dev/src/UI/Areas/Identity/Pages/Account/Manage/GenerateRecoveryCodes.cshtml#L23

**Comment by pranavkm** [comment]
@natemcmaster aside from merging this change in, do I need to do anything to produce a build for this?

**Comment by natemcmaster** [comment]
Yup. I'll talk to you offline as it involves merging to private forks

**Pull request merged**